{{cookiecutter.project_name}}/CONTRIBUTING.md

```diff
@@ -34,72 +32,69 @@ The best way to get your bug fixed is to provide a test case,
 and/or steps to reproduce the issue.
 
 
-How to request a feature
-------------------------
+# How to request a feature
 
-Request features on the `Issue Tracker`_.
+Request features on the [Issue Tracker][].
 
 
-How to set up your development environment
-------------------------------------------
+# How to set up your development environment
+
 
 You need Python 3.7+ and the following tools:
 
-- Poetry_
-- Nox_
-- nox-poetry_
+- [Poetry][]
+- [Nox][]
+- [nox-poetry][]
 
 Install the package with development requirements:
 
-.. code:: console
-
+```bash
    $ poetry install
```

Comment:
Per @paw-lu should you remove the prepending white space in the various places in this doc?

Reply:
Same as mentioned above. 

Current content:

Install the package with development requirements:

\```console
$ poetry install
\```